app/views/assignments/edit.haml

```diff
@@ -1,12 +1,23 @@
 %h1.title Edit assignment
 = form_for @assignment, url: roster_assignment_url(@roster, @assignment) do |f|
-  = f.label :start_date
-  = f.text_field :start_date, class: 'datepicker'
-  = f.label :end_date
-  = f.text_field :end_date, class: 'datepicker'
-  = f.label :user_id
-  = f.select :user_id, @users.pluck(:last_name, :id)
-  = f.submit 'Save'
-
-= button_to 'Delete assignment', roster_assignment_path(@roster, @assignment),
-  method: :delete
+  .container-fluid
+    .row
+      .col-xs-3
+      .col-xs-2
```

Comment:
I think you should be able to ditch the leading `.col-xs-3` in favor of making this `.col-xs-2.col-xs-offset-3`.